misc/e2e-aws.sh

```diff
@@ -223,11 +212,13 @@ stage-destroy() {
   cd ${BUILDDIR}
 
   source envs.sh
+
   terraform init ../platforms/aws/giantnetes
   terraform destroy -force ../platforms/aws/giantnetes
 
+  # Remove manually the bucket because it should stay in a prod env
```

Comment:
If it created by terraform it should be destroyed by terraform. No need to destroy it manually

Reply:
True now it should be done by Terraform